src/plugin/scoring-similarity/src/java/org/apache/nutch/scoring/similarity/cosine/Model.java

```diff
@@ -115,6 +126,7 @@ public static DocVector createDocVector(String content) {
       tStream.reset();
       while(tStream.incrementToken()) {
         String term = charTermAttribute.toString();
+        LOG.info(term);
```

Comment:
This seems like its used for debugging, please change it to LOG.debug(). It helps keeping the log clean. 
Thanks!